cli/ssh_test.go

```diff
@@ -86,6 +91,62 @@ func TestSSH(t *testing.T) {
 		pty.WriteLine("exit")
 		<-cmdDone
 	})
+	t.Run("StartStoppedWorkspace", func(t *testing.T) {
+		t.Parallel()
+
+		//store, ps := dbtestutil.NewDB(t)
+		//ownerClient := coderdtest.New(t, &coderdtest.Options{Pubsub: ps, Database: store, IncludeProvisionerDaemon: true})
+		//ownerClient.SetLogger(slogtest.Make(t, nil).Named("client").Leveled(slog.LevelDebug))
+		//first := coderdtest.CreateFirstUser(t, ownerClient)
+		//client, user := coderdtest.CreateAnotherUser(t, ownerClient, first.OrganizationID)
+		//
+		//// Create a stopped workspace.
+		//workspaceData := dbfake.WorkspaceBuild(t, store, database.Workspace{
+		//	OrganizationID: first.OrganizationID,
+		//	OwnerID:        user.ID,
+		//}).Seed(database.WorkspaceBuild{
+		//	Transition: database.WorkspaceTransitionStop,
+		//}).WithAgent().Do()
+		//workspace := workspaceData.Workspace
+		//
```

Comment:
Should we remove this or uncomment and remove the other?

```suggestion
```

Reply:
Whoops, I thought I removed this.

Reply:
I had a hard time getting this to work